### ISSUE TITLE
other organisms

### ISSUE THREAD

**Issue by zachflam** [shahpr/RiboViz#33]
Do you hope to expand your analysis and visualization scripts for profiling data from other organisms?

**Comment by ewallace** [comment]
Yes, we are writing a grant right now to do that. Which organisms and which aspects of the analysis/visualization are you most interested in?

**Comment by shahpr** [comment]
The analysis scripts have been expanded for human and mouse datasets. You can generate the necessary GFF and FASTA files using [this](https://github.com/shahpr/RiboViz/blob/master/scripts/script_for_transcript_annotation.Rmd). For visualization, as Edward mentioned, we are actively working on it.

**Comment by ewallace** [comment]
Migrated to https://github.com/riboviz/RiboViz/issues/29 and https://github.com/riboviz/RiboViz/issues/75
